packages/core/components/Modal/DataSourcePrompt/index.tsx

```diff
@@ -0,0 +1,210 @@
+import {
+    ActionButton,
+    DefaultButton,
+    Icon,
+    Spinner,
+    SpinnerSize,
+    TextField,
+} from "@fluentui/react";
+import { debounce } from "lodash";
+import * as React from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { ModalProps } from "..";
+import BaseModal from "../BaseModal";
+import { interaction, metadata, selection } from "../../../state";
+
+import styles from "./DataSourcePrompt.module.css";
+
+interface Props extends ModalProps {
+    isEditing?: boolean;
+}
+
+const DATA_SOURCE_DETAILS = [
+    "The files must contain at least the following columns: 'file_name', 'file_size', 'file_path', 'file_id', and 'uploaded' & must be unique by the 'file_path' column. Any other columns are optional and will be available as annotations to query by.",
+    "The 'file_path' column should contain the full path to the file in the cloud storage. The 'file_id' column should contain a unique identifier for the file. The 'uploaded' column should contain the date the file was uploaded to the cloud storage.",
+    'Data source files can be generating by this application by selecting some files, right-clicking, and selecting the "Generate CSV Manifest" option.å',
```

Comment:
Is the weird a intentional?

Reply:
In same line, noticing 'generating' instead of 'generated'

Reply:
Fixed in https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/67/commits/42d30c40b1193950ddacd495ebd80647a66dda92